packages/blocks/src/__internal__/utils/selection.ts

```diff
@@ -414,7 +425,16 @@ export function handleNativeRangeDragMove(
   startRange: Range | null,
   e: SelectionEvent
 ) {
-  const isForward = e.x > e.start.x || e.y > e.start.y;
+  const startBlock = getBlockByPoint(e.start);
+  assertExists(startBlock);
+  const startRect = startBlock.getBoundingClientRect();
+
+  const smallDistance = 6;
```

Comment:
why 6?

Reply:
No particular criterion, any suggestion?

Reply:
I think, at least, make all `6` into a constant to declare that they are pointing to the same 6 under same meaning

Reply:
Thanks! Done, Let's make it 2px, which is also used in isFarEnough()